src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -963,6 +963,26 @@ public static unsafe SocketError SetSockOpt(SafeCloseSocket handle, SocketOption
                     return err == Interop.Error.SUCCESS ? SocketError.Success : GetSocketErrorForErrorCode(err);
                 }
             }
+            else if (optionLevel == SocketOptionLevel.IP)
+            {
+                if (optionName == SocketOptionName.MulticastInterface)
+                {
+                    /* if the value of the IP_MULTICAST_IF is an address in the 0.x.x.x block
+                       the value is interpreted as an interface index */
```

Comment:
Nit: I think elsewhere we've shied away from using ```/*...*/``` syntax across multiple lines, so maybe just use ```//``` syntax.